tex/chProgramming.tex

```diff
@@ -1375,7 +1386,7 @@ \subsection{The (polymorphic) sequence data type}
 
 
 The name \C{seq} refers to (finite) ``sequences'', also called
-``lists''. This definition actually describes the type
+``lists''. This definition describes the type
```

Comment:
I don't approve this change.

Reply:
What does the word "actually" mean for you? To me it means that we are making things more correct or more precise, so I don't understand its meaning in this sentence (there is nothing wrong about "sequences" or "lists").

Reply:
My point is only to avoid too many commas in short succession: as a reader I see a comma as a place to take your breath, and if you take your breath too often, you choke (in other words, avoiding too many commas keeps some musicality in the phrasing).  So, I would kept the comma if there was not already another one in the sentence.

Reply:
Sorry, one of my comment was misplaced.  I hope it is correct now.

Reply:
"actually" refers to the fact "seq" is only a synonym for "list".

The situation is not very satisfactory, because if we type "Print list." in Coq, we get the following text:

Inductive list (A : Type) : Type :=  nil : seq A | cons : A -> seq A -> seq A

So "list" is used in the first part, and "seq" is used after the ":=" characters.  This is messy at best.